packages/core/src/external/ehr/command/write-back/grouped-vitals.ts

```diff
@@ -46,11 +53,45 @@ function getEhrWriteBackGroupedVitalsHandler(ehr: EhrSource): WriteBackGroupedVi
   return handler;
 }
 
-export const writeBackGroupedVitalsEhrs = [EhrSources.elation];
+export const writeBackGroupedVitalsEhrs = [EhrSources.athena, EhrSources.elation];
 export function isWriteBackGroupedVitalsEhr(ehr: EhrSource): boolean {
   return writeBackGroupedVitalsEhrs.includes(ehr);
 }
 
 export function isGroupedVitalsByDate(val: unknown): val is GroupedVitalsByDate {
   return Array.isArray(val) && val.length === 2 && val[0] instanceof Date && Array.isArray(val[1]);
 }
+
+// TODO: after Thomas PR! Rebase and add logic for add group by code  & group date here
```

Comment:
Agreed to wait on @thomasyopes changes from upstream for me to rebase and improve this logic

Reply:
done